### ISSUE TITLE
Please make a flatpak app for Elementary OS 6

### ISSUE THREAD

**Issue by iguys** [artemanufrij/webpin#147]
Please make a flatpak app for Elementary OS 6 , we are really missing this app in the appstore

**Comment by dcowan-london** [comment]
@iguys I'm currently working on updating WebPin for eOS6.

You can follow the instructions [here](https://github.com/dcowan-london/webpin/tree/flatpak#with-flatpak) to test my current version.

At the third stage (`git clone`), use this instead:
```
git clone https://github.com/dcowan-london/webpin.git
cd webpin
git checkout flatpak
```

You can see my updates at #143

**Comment by iguys** [comment]
Getting this error while builder the flatpak : flatpak-builder build com.github.artemanufrij.webpin.yml --user --install --force-clean
error: io.elementary.Sdk/x86_64/6.1 not installed
Failed to init: Unable to find sdk io.elementary.Sdk version 6.1

**Comment by iguys** [comment]
Trying to build flatpak but getting this error : flatpak-builder build com.github.artemanufrij.webpin.yml --user --install --force-clean
error: io.elementary.Sdk/x86_64/6.1 not installed
Failed to init: Unable to find sdk io.elementary.Sdk version 6.1

I also tried to install elementary SDK 
Getting Error : sudo apt install elementary-sdk
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 elementary-sdk : Depends: libgala-dev but it is not going to be installed
                  Recommends: io.elementary.vala-lint but it is not installable
E: Unable to correct problems, you have held broken packages.

**Comment by dcowan-london** [comment]
Are you running the latest version of elementary? What's the output of `lsb_release -a`?

Try a `sudo apt update && sudo apt upgrade`, then try installing the SDK again.

**Comment by dcowan-london** [comment]
@iguys Did that work?

**Comment by dcowan-london** [comment]
@iguys Turns out you don't actually need to install `elementary-sdk` - just go from the Flatpak heading on the "Install elementaryOS developer enviroment" page

**Comment by iguys** [comment]
Yes I was on 6.1 but after upgrading latest update, I was able to install elementary-sdk this morning. But after installing sdk my system was broken and I restored it by running my latest timeshift snapshot. Now I also think that it is not good idea to install the elementary-sdk. I will try using your suggested method without installing sdk very soon. Thanks
